generator/src/main/java/org/corfudb/generator/Generator.java

```diff
@@ -163,11 +171,23 @@ public void runLongevityTestForVerification() {
         consumeOperationQueue();
         runCpTrimOperation();
 
+        boolean interrupted = false;
+
         while (true) {
+
+            if (interrupted) {
+                break;
+            }
             // One round of 'chunk' starts
             long cumulativeRuntime = 0;
             while (true) {
                 Sleep.sleepUninterruptibly(POLL_TIME);
+
+                interrupted = Thread.currentThread().isInterrupted();
```

Comment:
The sleepUninterruptibly method will throw an UnrecoverableCorfuInterruptedError if interrupted, so you can remove all the `interrupted` checks.

Reply:
@xnull since Wenbin has pointed out, I'll remove this.